formats/GG.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Description": "GG",
+  "RedundantCharacters": " -",
+  "ValidationRegex": "^GG[0-9]{2,3}[A-Za-z]{2}$",
+  "TestData": {
+    "Valid": [
+      "GG1 1AA",

```

Comment:
It seems like postal codes start with "GY" instead of GG: https://worldpostalcode.com/guernsey/guernsey-channel-islands

Reply:
You are right. It has been fixed.